src/app.ts

```diff
@@ -41,6 +41,7 @@ if (config.START_WRITERS) {
   system.txOffsetRepairWorker.start();
   system.txRepairWorker.start();
   system.bundleRepairWorker.start();
+  system.mempoolWatcher?.start();
```

Comment:
I think it make sense to just start it if `config.START_WRITERS` is `true`, right?